dbt_subprojects/daily_spellbook/models/xrpl/gas_xrpl_fees.sql

```diff
@@ -0,0 +1,169 @@
+{{
+    config(
+        schema = 'gas_xrpl',
+        alias = 'fees',
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['block_date','unique_key'],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_date')],
+        post_hook='{{ expose_spells(\'["xrpl"]\',
+                                    "sector",
+                                    "gas",
+                                    \'["krishhh"]\') }}'
+    )
+}}
+
+WITH xrp_prices AS (
+    SELECT
+        DATE_TRUNC('minute', minute) AS price_minute
+        ,AVG(price) AS price_usd
+    FROM {{ source('prices', 'usd') }}
+    WHERE symbol = 'XRP' 
+        AND blockchain IS NULL
+    GROUP BY 1
+),
+xrpl_gas_fees AS (
+    SELECT 
+        'xrpl' AS blockchain
+        ,CAST(DATE_TRUNC('month', CAST(ledger_close_date AS TIMESTAMP)) AS DATE) AS block_month
+        ,CAST(DATE_TRUNC('day', CAST(ledger_close_date AS TIMESTAMP)) AS DATE) AS block_date
+        ,CAST(ledger_close_date AS TIMESTAMP) AS block_time
```

Comment:
ignoring time columns based on your comment. expectation is data output should match our standard EVM month/date/time columns.